modules/fts/pages/fts-advanced-store-dynamic-field.adoc

```diff
@@ -0,0 +1,5 @@
+= Store Dynamic Fields
+When checked, ensures inclusion of field-content in returned results.
```

Comment:
Not true.

1. You will still need to ask for the specific field content while querying
2. This is also needed for highlighting dynamic fields
3. Need to talk about how this could affect the index size